source/base/quadrature_selector.cc

```diff
@@ -45,7 +45,7 @@ QuadratureSelector<dim>::create_quadrature(const std::string &s,
       else if (s == "simpson")
         return QSimpson<dim>();
       else if (s == "trapez")
```

Comment:
shouldn't we add "trapezoidal" here as well? Also please leave a comment DEAL_II_DEPRECATE so we can grep and remove later.

Reply:
can you leave a comment here that contains "DEAL_II_DEPRECATED" so we can grep for this in the future?

Reply:
So happened!